2023-11-30 16:42:08,961 INFO    MainThread:12176 [wandb_setup.py:_flush():68] Configure stats pid to 12176
2023-11-30 16:42:08,961 INFO    MainThread:12176 [wandb_setup.py:_flush():68] Loading settings from /home/paperspace/.config/wandb/settings
2023-11-30 16:42:08,961 INFO    MainThread:12176 [wandb_setup.py:_flush():68] Loading settings from /home/paperspace/Documents/melanoma_image_classification/notebooks/wandb/settings
2023-11-30 16:42:08,961 INFO    MainThread:12176 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-11-30 16:42:08,961 INFO    MainThread:12176 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-11-30 16:42:08,961 INFO    MainThread:12176 [wandb_init.py:_log_setup():476] Logging user logs to /home/paperspace/Documents/melanoma_image_classification/notebooks/wandb/run-20231130_164208-1v4qfo13/logs/debug.log
2023-11-30 16:42:08,962 INFO    MainThread:12176 [wandb_init.py:_log_setup():477] Logging internal logs to /home/paperspace/Documents/melanoma_image_classification/notebooks/wandb/run-20231130_164208-1v4qfo13/logs/debug-internal.log
2023-11-30 16:42:08,962 INFO    MainThread:12176 [wandb_init.py:_jupyter_setup():426] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fcaec93f250>
2023-11-30 16:42:08,962 INFO    MainThread:12176 [wandb_init.py:init():516] calling init triggers
2023-11-30 16:42:08,962 INFO    MainThread:12176 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2023-11-30 16:42:08,962 INFO    MainThread:12176 [wandb_init.py:init():569] starting backend
2023-11-30 16:42:08,962 INFO    MainThread:12176 [wandb_init.py:init():573] setting up manager
2023-11-30 16:42:08,965 INFO    MainThread:12176 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-11-30 16:42:08,966 INFO    MainThread:12176 [wandb_init.py:init():580] backend started and connected
2023-11-30 16:42:08,974 INFO    MainThread:12176 [wandb_run.py:_label_probe_notebook():1113] probe notebook
2023-11-30 16:42:08,975 INFO    MainThread:12176 [wandb_run.py:_label_probe_notebook():1123] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-11-30 16:42:08,975 INFO    MainThread:12176 [wandb_init.py:init():658] updated telemetry
2023-11-30 16:42:09,034 INFO    MainThread:12176 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2023-11-30 16:42:09,323 INFO    MainThread:12176 [wandb_run.py:_on_init():2000] communicating current version
2023-11-30 16:42:09,387 INFO    MainThread:12176 [wandb_run.py:_on_init():2004] got version response upgrade_message: "wandb version 0.16.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-11-30 16:42:09,387 INFO    MainThread:12176 [wandb_init.py:init():728] starting run threads in backend
2023-11-30 16:42:09,748 INFO    MainThread:12176 [wandb_run.py:_console_start():1980] atexit reg
2023-11-30 16:42:09,748 INFO    MainThread:12176 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.WRAP_RAW
2023-11-30 16:42:09,750 INFO    MainThread:12176 [wandb_run.py:_redirect():1903] Wrapping output streams.
2023-11-30 16:42:09,750 INFO    MainThread:12176 [wandb_run.py:_redirect():1925] Redirects installed.
2023-11-30 16:42:09,751 INFO    MainThread:12176 [wandb_init.py:init():765] run started, returning control to user process
2023-11-30 16:44:11,660 INFO    MainThread:12176 [wandb_init.py:_pause_backend():389] pausing backend
2023-11-30 16:44:11,661 INFO    MainThread:12176 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2023-11-30 16:45:00,959 INFO    MainThread:12176 [wandb_init.py:_resume_backend():398] resuming backend
2023-11-30 16:45:00,962 INFO    MainThread:12176 [wandb_init.py:_pause_backend():389] pausing backend
2023-11-30 16:45:00,963 INFO    MainThread:12176 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2023-11-30 16:45:01,080 INFO    MainThread:12176 [wandb_init.py:_resume_backend():398] resuming backend
2023-11-30 16:45:01,544 INFO    MainThread:12176 [wandb_setup.py:_flush():68] Configure stats pid to 12176
2023-11-30 16:45:01,545 INFO    MainThread:12176 [wandb_setup.py:_flush():68] Loading settings from /home/paperspace/.config/wandb/settings
2023-11-30 16:45:01,545 INFO    MainThread:12176 [wandb_setup.py:_flush():68] Loading settings from /home/paperspace/Documents/melanoma_image_classification/notebooks/wandb/settings
2023-11-30 16:45:01,545 INFO    MainThread:12176 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-11-30 16:45:01,545 INFO    MainThread:12176 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-11-30 16:45:01,545 INFO    MainThread:12176 [wandb_init.py:_log_setup():476] Logging user logs to /home/paperspace/Documents/melanoma_image_classification/notebooks/wandb/run-20231130_164501-2sx2hftr/logs/debug.log
2023-11-30 16:45:01,545 INFO    MainThread:12176 [wandb_init.py:_log_setup():477] Logging internal logs to /home/paperspace/Documents/melanoma_image_classification/notebooks/wandb/run-20231130_164501-2sx2hftr/logs/debug-internal.log
2023-11-30 16:45:01,545 INFO    MainThread:12176 [wandb_init.py:init():516] calling init triggers
2023-11-30 16:45:01,545 INFO    MainThread:12176 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2023-11-30 16:45:01,545 INFO    MainThread:12176 [wandb_init.py:init():535] re-initializing run, found existing run on stack: 1v4qfo13
2023-11-30 16:45:01,553 INFO    MainThread:12176 [wandb_run.py:_finish():1746] finishing run yashchks87/melanoma-classification/1v4qfo13
2023-11-30 16:45:01,554 INFO    MainThread:12176 [jupyter.py:save_history():448] not saving jupyter history
2023-11-30 16:45:01,554 INFO    MainThread:12176 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2023-11-30 16:45:01,554 INFO    MainThread:12176 [wandb_init.py:_jupyter_teardown():408] cleaning up jupyter logic
2023-11-30 16:45:01,554 INFO    MainThread:12176 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2023-11-30 16:45:01,554 INFO    MainThread:12176 [wandb_run.py:_restore():1932] restore
2023-11-30 16:45:01,554 INFO    MainThread:12176 [wandb_run.py:_restore():1938] restore done
2023-11-30 16:45:06,675 INFO    MainThread:12176 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2023-11-30 16:45:06,676 INFO    MainThread:12176 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2023-11-30 16:45:06,681 INFO    MainThread:12176 [wandb_run.py:_footer_sync_info():3333] logging synced files
